t/t1210-pull.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+#
+# Copyright (c) 2006 Yann Dirson
+#
+
+test_description='test
+
+'
```

Comment:
Need better description.

Also, `t1210-pull.sh` is a bit too generic considering there are already four other "pull" test scripts. Please use a more descriptive name for this script.

Reply:
Agreed, added the description, I missed that field. 

I renamed the test to be more descriptive, though I named it generically just pull because it was testing a generic pull scenario, seems like pull command does not have a generic test to contain common testcases.